lib/listen.js

```diff
@@ -18,7 +19,17 @@ module.exports = function decorate(Class) {
       .on('read', this.read.bind(this))
       .on('update', this.update.bind(this))
       .on('delete', this.delete.bind(this))
-      .on('list', this.list.bind(this));
+      .on('list', this.list.bind(this))
+      .on('deleteAll', this.deleteAll.bind(this))
```

Comment:
I would advise not including `deleteAll` and `reset` topics into the store by default. 

These are pretty destructive and it should be the responsibility of the application developer to decide to include a topic this destructive.